python/lsst/daf/butler/datastores/fileDatastore.py

```diff
@@ -882,7 +883,8 @@ def _prepIngest(self, *datasets: FileDataset, transfer: Optional[str] = None) ->
     def _finishIngest(self, prepData: Datastore.IngestPrepData, *, transfer: Optional[str] = None) -> None:
         # Docstring inherited from Datastore._finishIngest.
         refsAndInfos = []
-        for dataset in prepData.datasets:
+        progress = Progress("daf.butler.datastores.FileDatastore.ingest")
```

Comment:
in daf_butler we always use `lsst.daf.butler` in logger names because we were hoping that people outside of LSST would be interested and where it might be confusing for the `lsst` to be implicit.